setup-dev.sh

```diff
@@ -5,6 +5,24 @@ echo "# Install testing tools."
 echo "# Only works with Ubuntu / APT."
 echo "# --------------------------------------"
 
+hookScriptName=".git/hooks/pre-push"
+
+if [ -t 1 ]; then
+    echo "Install a git pre-push hook?"
+    select yn in "Yes" "No"; do
```

Comment:
Let's just check if the input starts with "y", in all other cases we don't do anything, so you can convert this to a single if statement.